modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -127,129 +156,98 @@ endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
 .Procedure
 
-ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
-. Set the `$RELEASE_IMAGE` variable by running the following command:
+. Set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
 +
 [source,terminal]
 ----
 $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
 ----
-endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 
 . Extract the list of `CredentialsRequest` objects from the {product-title} release image by running the following command:
 +
 [source,terminal]
-ifdef::aws-sts[]
-----
-$ oc adm release extract \
-  --credentials-requests \
-  --cloud=aws \
-  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
-  --from=quay.io/<path_to>/ocp-release:<version>
-----
-endif::aws-sts[]
-ifdef::google-cloud-platform[]
-----
-$ oc adm release extract \
-  --credentials-requests \
-  --cloud=gcp \
-  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
-  quay.io/<path_to>/ocp-release:<version>
-----
-endif::google-cloud-platform[]
-ifdef::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
 ----
 $ oc adm release extract \
+  --from=$RELEASE_IMAGE \
   --credentials-requests \
-  --cloud=alibabacloud \
-  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
-  $RELEASE_IMAGE
+  --included \// <1>
+  --install-config=<path_to_directory_with_installation_configuration>/install-config.yaml \// <2>
+  --to=<path_to_directory_for_credentials_requests> <3>
 ----
-endif::alibabacloud-default,alibabacloud-customizations,alibabacloud-vpc[]
-+
-<1> `credrequests` is the directory where the list of `CredentialsRequest` objects is stored. This command creates the directory if it does not exist.
+<1> The `--included` parameter includes only the manifests that your specific cluster configuration needs.
+<2> Specify the location of the `install-config.yaml` file.
+<3> Specify the path to the directory where you want to store the `CredentialsRequest` objects. If the specified directory does not exist, this command creates it.
 +
 [NOTE]
 ====
 This command can take a few moments to run.
 ====
 
-ifdef::aws-sts[]
-. If your cluster uses cluster capabilities to disable one or more optional components, delete the `CredentialsRequest` custom resources for any disabled components.
-+
-.Example `credrequests` directory contents for {product-title} 4.12 on AWS
+. Optional: Review the contents of the `credrequests` directory. Some of the following `CredentialsRequest` CRs are for components that are optional and might be disabled in your cluster.
```

Comment:
> Some of the following `CredentialsRequest` CRs are for components that are optional and might be disabled in your cluster.

Can we drop this sentence?  `--included` should have already filtered out anything the coming cluster isn't expected to need.  I have no problem with:

> Optional: Review the contents of the `credrequests` directory.

just to increase transparency before creating credentials, but I don't think we need to hint that users might need to perform their own filtering anymore.

Reply:
Oh - I was thinking of it as "just in case you are wondering why you don't see XYZ CR that is listed in this example output. Do you think maybe with this update we should just drop the whole step?

Reply:
ah, I see what you were saying now.  Maybe drop it here, and, if we use an optional CR later, call out the optionality when introducing that example CR?